### PR TITLE
libharu 2.4.3

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Cherry-picked @chenrui333 commit from https://github.com/Homebrew/homebrew-core/pull/109718

File: Formula/harbour.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Harbour < Formula
   desc "Portable, xBase-compatible programming language and environment"
   homepage "https://harbour.github.io"
   license "GPL-2.0"
-  revision 1
+  revision 2
```
Comment (cho-m) [inline Formula/harbour.rb:5]: I rebuilt `harbour` a couple days ago (#107019) so can probably drop commit, rebase, and retry

Comment (chenrui333) [reply Formula/harbour.rb:5]: 👍

**Comment by chenrui333** [comment]
bump to v2.4.3 and rebased to the latest master

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**